irohad/consensus/yac/impl/yac_gate_impl.cpp

```diff
@@ -42,8 +42,9 @@ namespace iroha {
             block_loader_(std::move(block_loader)),
             delay_(delay) {
         log_ = logger::log("YacGate");
-        block_creator_->on_block().subscribe(
-            [this](auto block) { this->vote(block); });
+        block_creator_->on_block().subscribe([this](auto block) {
```

Comment:
one block copy

Reply:
block is `shared_ptr` passing by value is acceptable and more thread-safe